.swiftlint.yml

```diff
@@ -31,6 +32,8 @@ disabled_rules:
   - overridden_super_call
   - redundant_string_enum_value
   - no_fallthrough_only
+  - class_delegate_protocol
```

Comment:
Disabled because it fails for @objc protocols inheriting NSObjectProtocol, not completely sure if we want to enable it afterwards